src/lib/maplibre/layer_helper_utils.ts

```diff
@@ -0,0 +1,37 @@
+import { get } from "svelte/store";
+import { map as mapStore } from "../../stores";
+import { LayerHelper } from "./layer_helper";
+
+export const roadLayerIds = [
```

Comment:
```suggestion
// These depend on MapTiler basemap styles
export const roadLayerIds = [
```

I see some footpaths missing, we can dig through later and improve this set